back/src/domains/convention/use-cases/ResyncOldConventionsToFt.unit.test.ts

```diff
@@ -58,394 +58,717 @@ describe("ResyncOldConventionsToPe use case", () => {
     );
   });
 
-  describe("Right paths", () => {
-    it("broadcast two conventions to pe", async () => {
-      uow.agencyRepository.agencies = [toAgencyWithRights(agencyPE)];
-      uow.conventionRepository.setConventions([
-        conventionToSync1,
-        conventionToSync2,
-      ]);
-      uow.conventionsToSyncRepository.setForTesting([
-        {
-          id: conventionToSync1.id,
-          status: "TO_PROCESS",
-        },
-        {
-          id: conventionToSync2.id,
-          status: "TO_PROCESS",
-        },
-      ]);
-      expectToEqual(ftGateway.legacyBroadcastConventionCalls, []);
-
-      const report = await useCase.execute();
-
-      expectToEqual(uow.conventionRepository.conventions, [
-        conventionToSync1,
-        conventionToSync2,
-      ]);
-      expectToEqual(uow.conventionsToSyncRepository.conventionsToSync, [
-        {
-          id: conventionToSync1.id,
-          status: "SUCCESS",
-          processDate: timeGateway.now(),
-        },
-        {
-          id: conventionToSync2.id,
-          status: "SUCCESS",
-          processDate: timeGateway.now(),
+  it("also broadcast assessment if present", async () => {
```

Comment:
Dans les right paths ?

Reply:
C'est à supprimer, il est déjà dans les rights paths

Reply:
c'est à supprimer, il est déjà dans les rights paths de "when feature to standard format for convention broadcast is ON"